### PR TITLE
Assorted Fixes

### LINKED ISSUE #17: Remove duplicate entries on the client
So the `tracking_ids` are not actually stable / consistent. Steam can assign 2 ids to the same game. see #5. 

Fixing on the indexing level is too much work and will spam steamdb's servers more than I'd want to. Just dedup exact title matches on the frontend.

### LINKED ISSUE #19: Remove genre select, just adds extra noise and isn't very helpful. (More filters also slows the search down)

### LINKED ISSUE #20: Lower page size and add pagination.
Lower the page size so that recomendations and pages feel more side by side

### LINKED ISSUE #21: optional: Add open in steamdb link
Going to try to ask permission from the site admins, but could be nice to give credit to the creators.

### LINKED ISSUE #22: add `totalPositiveNegative` slider

### LINKED ISSUE #23: tracking issue: wrapping up steamdb
Overall, thinking to have 2 sliders below the search bar pos/neg vote ratio and total pos/neg votes. With the default being the values I set in #15. From attending the discord, people really seem to enjoy the :+1: and :-1: voting mechanism over the metacritic score or even steamdb's builtin scoring function. Some very popular games don't even have metacritic scores.

A few attempts here are to simplify the UI, we mainly want to just have a simple streamlined experience for people wanting to look for games for ~2-5 minutes.

- [x] #28
- [x] #27
- [x] #26
- [x] #25
- [x] #24
- [x] #21 
- [x] #16 
- [x] #17 
- [x] #20 
- [x] #19
- [x] #22 
- [x] #18

### LINKED ISSUE #24: Show how many positive and negative recomendations you chose underneath "Recommended Games"
Thinking this would be something like "Out of 10 likes and 5 dislikes" underneath this
image

### LINKED ISSUE #26: Way to view all the things voted on
Two options for how this looks:
- The component we had before' was very good for this, if the site is less scrollable it shouldn't be too invasive.
- A button in the corner that has a modal may just be easier and less to worry about. I trust your judgement.

### LINKED ISSUE #27: Storing Votes in localstorage
Its very frustrating when the page refreshes the whole site has to re-render the page view.

### LINKED ISSUE #28: stretch goal: storing search state/filters in query parameters

### PR DESCRIPTION
**Pull request by SaraVieira**
- [x] #28
- [x] #27
- [x] #26
- [x] #25
- [x] #24
- [x] #21 
- [x] #16 
- [x] #17 
- [x] #20 
- [x] #19
- [x] #22 
- [x] #18 
- [x] Scroll to top on page change
- [x] Fix TS

### CODE REVIEW

File: ui/src/lib/gameState.ts [hunk 1]
```diff
@@ -1,154 +1,190 @@
...
+  getItem: async (name: string): Promise<string | null> => {
+    return (await get(name)) || null;
+  },
+  setItem: async (name: string, value: string): Promise<void> => {
+    await set(name, value);
+  },
+  removeItem: async (name: string): Promise<void> => {
+    await del(name);
+  },
+};
```
Comment (SaraVieira) [inline ui/src/lib/gameState.ts:24]: My amount of steam games was too much for local storage so this uses indexeddb

**Comment by cdxker** [comment]
I'm getting a 500 error with my steamid. Not sure if its a permission issue or if I'm doing the wrong steam account ID
image

Steam ID: 76561198310187678

**Comment by cdxker** [comment]
Good addition on adding the mobile view, was thinking to punt on that but since its here, can we make the breakpoint for mobile a little bit earlier.

**Comment by cdxker** [comment]
I'm seeing a weird issue between page refreshes / reloads also.

State_Being_Reset_On_Search

**Comment by SaraVieira** [comment]
all issues fixed and also added some basic CI

**Pull request merged**